events/system/interactionCreate.js

```diff
@@ -12,9 +13,9 @@ export default {
     }
 
     try {
-      const { userService } = interaction.services;
-      const isUser = await userService.getOneDiscordId(interaction.user.id);
-      if (!isUser) {
+      const { userService, sessionService } = interaction.services;
+      const dbUser = await userService.getOneDiscordId(interaction.user.id);
+      if (!dbUser) {
```

Comment:
I think user would be better here
```suggestion
      const user = await userService.getOneDiscordId(interaction.user.id);
      if (!user) {
```